### DISCUSSION TITLE
Title

### DISCUSSION THREAD

**Discussion by jeromekelleher** [tskit-dev/what-is-an-arg-paper#30] [General]
Following up on #20, the current "What is an ARG" title isn't quite capturing what we want the paper to be, or the core message that we want to convey to readers. It's also less likely to "hook in" the people we most want to read it, those who are already very familiar with ARGs.

So, let's break this down. What are we actually doing in the paper? I think the key thing is that we're (a) showing that the way we encode recombination breakpoint information is important, both computationally and in terms of what can be inferred; and (b) offering some terminology to allow us to discuss nuances that aren't really there at the moment---i.e., when do we care about non-coalescence coalescence nodes (need a better name, #23 )? 

I think the key idea is that we're looking "into" ARGs in a new level of detail. 

Some ideas (some are stupid):

1. The anatomy of an Ancestral Recombination Graph
2. How we encode recombination breakpoints in an Ancestral Recombination Graph is important
3. Storing breakpoint information on edges transforms the Ancestral Recombination Graph
4. Cracking open the Ancestral Recombination Graph

Something like (3) seems like the most likely title to grab our target audiences attention?

**Comment by hyanwong** [comment]
FWIW I was chatting to Alwyn and he says that he always thinks of an ARG as edge annotated anyway (and pointed out that Wiuf & Hein use edge annotations in places). I said I thought most people still thought in node-annotation terms, but he (slightly) pushed back against that interpretation.

I wonder if the gARG / eARG distinction is more fundamental anyway? So an alternative to 3. would be "An explicit representation of genomes transforms the Ancestral Recombination Graph". Or similar phrasings such as "Focussing on genomes transforms the ARG". Other than that, I quite like 1. but take your point about grabbing the right people's attention.

**Comment by JereKoskela** [comment]
In a vacuum I prefer 1. as well, but I take the point about a more specific title being more attention-grabbing. I think 3. would be improved if we can sneak in some indication of how the ARG is transformed. "Storing breakpoint information on edges facilitates scalable processing of Ancestral Recombination Graphs" is the best example of what I mean that springs to mind, though it also feels a bit narrow in its scope so there's probably room for improvement.

**Reply by hyanwong** [reply]
I think that the gARG necessitates edge annotations, but edge annotations do not necessarily necessitate gARGs. So I wonder if focussing on edge annotations is the right thing? OTOH edge annotations are the feature that allows scalability, and I completely see your point about mentioning that.

**Comment by jeromekelleher** [comment]
Yan and I just did a bit of brainstorming, how about 

- A general and efficient representation of Ancestral Recombination Graphs

**Reply by JereKoskela** [reply]
Sounds great to me!

**Reply by hyanwong** [reply]
Great. I'll change it shall I?

**Reply by jeromekelleher** [reply]
Of course we *could* also do "Succinct tree sequences: a general and efficient representation of Ancestral Recombination Graphs"

**Reply by hyanwong** [reply]
Worth considering, I think. I wonder if such an explicit plug would put anyone off, though?

**Reply by JereKoskela** [reply]
I'm happy to include the plug if it might be helpful, but aesthetically I prefer the shorter title. There's also a slight risk that putting "Succinct tree sequences" in the title makes it sound like we're only representing a sequence of marginal trees, even if the second half of the title already argues against that.

**Reply by jeromekelleher** [reply]
I think Jere is probably right, let's keep it the way it is.

**Reply by hyanwong** [reply]
Shall we close this then? - (edit - I forgot this is a discussion, and those don't get closed!)
